src/Bundle/test/app/config/config.yml

```diff
@@ -32,7 +32,11 @@ doctrine:
                 auto_mapping: true
 
 fos_rest:
+    body_listener:
+        enabled: true
     view:
+        view_response_listener:
+            enabled: true
```

Comment:
Are those config changes required? If so, we should mention them in UPGRADE.md.

Reply:
Yes it's required. I have to check if both are required.